coreweave/cks/resource_cluster.go

```diff
@@ -202,6 +210,31 @@ func (c *ClusterResourceModel) Set(cluster *cksv1beta1.Cluster) {
 			internalLbCidrs = append(internalLbCidrs, types.StringValue(c))
 		}
 		c.InternalLBCidrNames = types.SetValueMust(types.StringType, internalLbCidrs)
+		if !nodePortEmpty(cluster.Network.ServiceNodePortRange) {
+			c.NodePortRange = types.ObjectValueMust(
+				map[string]attr.Type{
+					"start": types.Int32Type,
+					"end":   types.Int32Type,
+				},
```

Comment:
I think this can use `c.NodePortRange.Attributes()` instead of configuring this here, but that's not a big deal